server/models/Volunteer.ts

```diff
@@ -0,0 +1,34 @@
+import { model, models, Model, Schema, Document } from "mongoose";
+import { Volunteer } from "utils/types";
+import EventSchema from "server/models/Event";
```

Comment:
Not sure if this compiles since the tests are showing errors. Here, you're importing the schema model, whereas you need the schema. Can you try `import { EventSchema } `?

Reply:
That was it, thanks! Still learning, so I appreciate the help!